charts/schema-registry/templates/configmap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "schema-registry.configmap" . }}
+data:
+  "CONTROLLER_URL": "tcp://{{ .Values.controller.ip }}:{{ .Values.controller.port }}"
+  "AUTHORIZATION_ENABLED": "{{ .Values.authentication.enabled }}"
+  "TLS_ENABLED": "{{ .Values.tls.enabled }}"
+  {{- if .Values.authentication.enabled }}
+  "USER_PASSWORD_FILE": "{{ include "auth-volume.path" . }}/{{ .Values.authentication.userPasswordFile }}"
```

Comment:
You are probably referring to Password Auth Handler user account database here. Please give it a more appropriate name.

Reply:
Curious, have we tested this changed? Does it work?